server/client.go

```diff
@@ -451,10 +451,11 @@ func (c *client) initClient() {
 	c.echo = true
 
 	c.debug = (atomic.LoadInt32(&c.srv.logging.debug) != 0)
-	c.trace = (atomic.LoadInt32(&c.srv.logging.trace) != 0)
 
-	if c.kind == SYSTEM && !c.srv.logging.traceSystemAcc {
-		c.trace = false
+	if c.kind == SYSTEM {
+		c.trace = c.srv.logging.traceSystemAcc
```

Comment:
Still like shorter traceSysAcc

Reply:
was outdated version. renamed it to traceSysAcc